libs/providers/go-feature-flag/src/lib/go-feature-flag-provider.ts

```diff
@@ -10,6 +10,7 @@ import {
   TypeMismatchError,
 } from '@openfeature/js-sdk';
 import axios from 'axios';
+import _ from 'lodash';
```

Comment:
I _think_ it's pure luck that this works, getting this transitively though another dep.

We don't have `lodash` as a dependency (though we do have the modularized [lodash.isEqual](https://www.npmjs.com/package/lodash.isequal) package) to keep bundle size small.

Reply:
oh good catch, my IDE worked on this one so I trust it _(I should be more careful the next time)_.